lib/tasks/cleanup_helper.rb

```diff
@@ -0,0 +1,72 @@
+module CleanupHelper
+  def clean_foreman_docker
+    perms = Permission.where(:resource_type => [:DockerRegistry, :Container])
+    perms.each { |perm| perm.filters.destroy_all }
+    perms.destroy_all
+  end
+
+  def clean_trends
+    ActiveRecord::Base.connection.drop_table(:trend_counters, if_exists: true)
+    ActiveRecord::Base.connection.drop_table(:trends, if_exists: true)
+    # Migration names: create_trends create_trend_counters add_range_to_trend_counters add_trend_counter_created_at_unique_constraint
+    ActiveRecord::SchemaMigration.where(version: %w[20121012170851 20121012170936 20150202094307 20181031155025]).delete_all
```

Comment:
I get you're just moving things around, but this is odd to me. The history is still there: it happened and can help explain the past. Rails doesn't care if a schema migration is in the DB but doesn't exist on disk.

Reply:
@ezr-ondrej do you recall why the `SchemaMigration` records needed to be deleted?

If it's not immediately obvious that we can delete it, I'll leave the two cleanup methods as-is.

Reply:
I'd be ok with that.

Reply:
sorry for late reply. We have moved the migrations under the same numbers to puppet plugin, so we need to remove them in order to be able to install the plugin again in the future.
This clean-up effectivelly undo those, so it seems natural to remove them from history (as it is basically roll back of those)

Reply:
Thanks for that information. I think that would have been a good code comment.